### PR TITLE
Net stuff, Scripts moved to fallback and delete empty scripts

### PR DESCRIPTION
**Pull request by Nickito12**
* Some net screen fixes(From updates to offline screens and lua 5.1 syntax)
* Delete empty lua files and make the installer delete til death
* Move some useful Scripts from Til Death to fallback

### CODE REVIEW

File: stepmania.nsi [hunk 1]
```diff
@@ -410,6 +410,7 @@ Section "Main Section" SecMain
 
 	; remove and install themes
 	RMDir /r "$INSTDIR\Themes\_fallback"
+	RMDir /r "$INSTDIR\Themes\Til Death"
```
Comment (martensm) [inline stepmania.nsi]: Can you make a separate commit for this? I don't want to do this here because it's something that needs to be discussed further.

File: Themes/Til Death/BGAnimations/ScreenEvaluation decorations/offsetplot.lua [hunk 2]
```diff
@@ -1 +0,0 @@
---moved to ../
```
Comment (martensm) [inline Themes/Til Death/BGAnimations/ScreenEvaluation decorations/offsetplot.lua]: Shouldn't this be in the previous commit?

File: Themes/Til Death/BGAnimations/ScreenEvaluation decorations/wifeScold.lua [hunk 3]
```diff
@@ -1 +0,0 @@
--- gone
```
Comment (martensm) [inline Themes/Til Death/BGAnimations/ScreenEvaluation decorations/wifeScold.lua]: Shouldn't this be in the previous commit?

**Comment by martensm** [comment]
There is too much going on in this PR, it needs to be more focused. If you just PR the scripts being moved to fallback changes then i'll merge it. I'd suggest creating a separate PR for each bullet point so they can be tracked more easier.

**Comment by Nickito12** [comment]
Ok, the net stuff wa re-PR'd in https://github.com/etternagame/etterna/pull/86
I'm gonna close this
